components/analytics/compact-dashboard.tsx

```
@@ -0,0 +1,383 @@
+    // Define time range options for the quick filters
+    const timeRanges = [
+        { value: "7d", label: "7 Days" },
+        { value: "14d", label: "14 Days" },
+        { value: "30d", label: "30 Days" },
+        { value: "90d", label: "90 Days" }
+    ]
+
+    // Handler for time range changes
+    const handleTimeRangeChange = (value: string) => {
+        const now = dayjs()
+        const days = parseInt(value.replace('d', ''))
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)